.github/workflows/eamxx-gh-ci-standalone.yml

```diff
@@ -0,0 +1,69 @@
+name: gh-standalone
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths:
+      # first, yes to these
+      - '.github/workflows/eamxx-gh-ci-standalone.yml'
+      - 'cime_config/**'
```

Comment:
Why would standalone be triggered by other components mods (including driver) or cime_config mods?

Reply:
Ah, I think cime_config mods may impact b/c of how you setup the env for ghci-oci in machine_specs.py. If that's the case, maybe we could narrow it down to `cime_config/machine/config_machines.xml`? Also, if what I'm saying is true, you should also trigger if there's a change in the cime submodule, since get_case_env may change (but I suppose we can assume chances of issues from this kind of changes are very slim)